snapper/scheduler.py

```diff
@@ -0,0 +1,62 @@
+from multiprocessing import Queue, Pool, Manager
+import shutil
+import os
+
+#from snapper import app
+from snapper.worker import host_worker
+
+import yaml
+from pathlib import Path
+
+
+#host_queue = None
+#pool = None
+
+class Scheduler:
+
+
+    def __init__(self):
+        with open(Path(__file__).parent / "config.yaml") as file:
```

Comment:
You don't need to do it here. Load it once inside some "config.py" and then import everywhere.

Reply:
i know, i tried import app and then app.config["etc"] and it gave keyword error

Reply:
Try again, maybe? It's either importing config in a separate file or using the current approach, but don't load it here anyway.